.github/workflows/NodeCI.yml

```diff
@@ -11,77 +11,82 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
       - uses: actions/setup-node@v3
       - name: Install Packages
-        run: yarn install
+        run: pnpm install
       - name: Lint
-        run: yarn lint
+        run: pnpm run lint
       - name: Build
-        run: yarn build
+        run: pnpm run build
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x, 18.x, 19.x]
+        node-version: [14.x, 16.x, 17.x, 18.x, 19.x]
```

Comment:
I dropped Node 12 because pnpm 7 doesn't support it.
Can we do this?

Reply:
How about creating a `test-for-node12` job and using pnpm v6?

```yml
  test-for-node12:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: pnpm/action-setup@v2
        with:
          version: 6.35.1
```

If it still doesn't work, let's drop support for node v12.
Since this package is still at 0.x, I believe it can contain breaking changes in minor versions.